Documentation/Features/NewFields/Index.rst

```diff
@@ -259,6 +259,15 @@ Configuration/Extbase/Persistence/Classes.php:
        \In2code\Femanagerextended\Domain\Model\User::class => [
            'tableName' => 'fe_users',
            'recordType' => 0,
+           'properties' => [
+               // add your added properties
+               'allowContact' => [
```

Comment:
@sbusemann 
Sorry! I just noticed like now that I have not adapted the Extbase property names!
it should be `'skypeId' => ` and `'twittterId' => `.

Reply:
Oh god, this went completely wrong. I'll do a new PR.